docs/database-functionalities/streams/kafka-streams.md

```diff
@@ -0,0 +1,167 @@
+---
+id: kafka-streams
+title: Managing Kafka streams
+sidebar_label: Managing Kafka streams
+---
+
+To connect Memgraph to a Kafka stream, we obviously need an existing stream.
+As Kafka itself is a quite complex system, we don't aim to to teach you about
+it. If you are not familiar with Kafka, then please check out their [quickstart
+guide](https://kafka.apache.org/quickstart) to get familiar with it. In the
+followings we assume that a Kafka server is available at on the 9092 port of
+thelocal machine ('localhost:9092`) as the default configuration of the Kafka
+quick start guide.
+
+## Configuring Memgraph
+
+As Memgraph can connect to only one Kafka cluster at once, the list of
+bootstrap servers can be set by the `--kafka-bootstrap-servers`
+configuration option. It has to be set to `localhost:9092` (or to an
+arbitrary address where a Kafka cluster is available).
+
+## Creating the stream
+
+The very first step is to make sure at least one transformation is loaded into
+Memgraph. If you are not sure, please check the regarding
+[documentation](/database-functionalities/streams/implement-transformation-module.md).
```

Comment:
@g-despot  I cannot make this work. Could you please check out what am I doing wrong?

Reply:
I don't what was to problem, but it is working now.